main/zoe/api/zoe-contract-facet.md

```diff
@@ -36,8 +36,8 @@ Create an empty offer for recordkeeping purposes (Autoswap uses this to create t
 let poolOfferHandle = zoe.escrowEmptyOffer();
```

Comment:
Let's remove the `## zcf.escrowEmptyOffer()` section too. See comment below on the recent change in Zoe 0.2.0 that replaced `zcf.escrowOffer` and `zcf.escrowEmptyOffer` with `zcf.getZoeService`